VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.ts

```diff
@@ -58,7 +58,9 @@ export class PrivateConsultationRoomControlsComponent extends HearingControlsBas
 
     get canJoinHearingFromConsultation(): boolean {
         return (
-            this.conferenceStatus?.newStatus === ConferenceStatus.InSession && this.participant?.status === ParticipantStatus.InConsultation
+            this.conferenceStatus?.newStatus === ConferenceStatus.InSession &&
+            this.participant?.status === ParticipantStatus.InConsultation &&
+            this.isHost
```

Comment:
should quick link participant be able to called into a hearing in session, I cant remember whether that was a requirement? This logic wouldn't impact that would it?

Reply:
I aren't sure but the button wont be visible as it is only  shown when they are available. So it shouldn't impact anything.

Reply:
They can be called in but they won't have a button to join.